### ISSUE TITLE
Request: CashFree

### ISSUE THREAD

**Issue by soubhatta** [simple-icons/simple-icons#12039]
### Brand Name

CashFree

### Website

https://www.cashfree.com/

### Popularity Metric

Global rank - 15889

### Official Resources for Icon and Color

https://www.cashfree.com/

### Additional Comments

Please Include this badge in your website - https://badges.pages.dev/

**Comment by PeterShaggyNoble** [comment]
It's not entirely clear from their [terms & conditions](https://www.cashfree.com/tnc/), but it looks like we may need permission to add this one.

**Comment by adamrusted** [comment]
I agree, the section `Contents of Cashfree Website` points C-E would indicate that we cannot include this without permission. Could you please reach out to them @soubhatta?

> Please Include this badge in your website - https://badges.pages.dev/

Also worth noting we do not maintain that page. That is owned by @developStorm.

**Comment by adamrusted** [comment]
Closing assuming no progress on permissions. @soubhatta - if you've heard from the brand, please let us know here.
